src/commands/fun.command.ts

```diff
@@ -0,0 +1,344 @@
+import { Command, CommandMessage, Guard, Description } from "@typeit/discord";
+import { NotBot } from "../guards/NotABot.guard";
+import { Logger } from "../services/logger.service";
+import NekoClient = require("nekos.life");
+import { COLOR } from "../enum/colors.enum";
+import { MessageEmbed } from "discord.js";
+import { Mention } from "../guards/Mention.guard";
+
+function getMentioned(command: CommandMessage) {
+    try {
+        const mentioned = command.mentions.users.first();
+        const guildMentioned = command.guild.member(mentioned);
+        return guildMentioned ? guildMentioned.nickname : guildMentioned.user.username;
+    } catch (error) {
+        Logger.prototype.error("Undefined user");
+        return "error";
+    }
+}
+export abstract class Fun {
```

Comment:
A lot of this code is duplicated, see if you can write a generic function in which you just pass in the command name?